client/jetpack-connect/authorize.js

```diff
@@ -771,6 +771,22 @@ export class JetpackAuthorize extends Component {
 			return redirectAfterAuth;
 		}
 
+		// We naviage users to complete page if it's not a multisite and feature flag activated
+		if ( config.isEnabled( 'jetpack/offer-complete-after-activation' ) ) {
+			const isMultisite = this.props.site && this.props.site.is_multisite;
+			if ( ! isMultisite && this.props.site ) {
+				const jpcTargetCompete = addQueryArgs(
+					{ redirect: redirectAfterAuth },
```

Comment:
I don't think we need to add in this `redirect` url query arg (`?redirect=whatever`) to the final redirection target, do we?

With it there, we redirect to: `http://calypso.localhost:3000/jetpack/connect/plans/complete/retail-cat.jurassic.ninja?redirect=https%3A%2F%2Fretail-cat.jurassic.ninja%2Fwp-admin%2Fadmin.php%3Fpage%3Djetpack  (notice the url query arg).

And without it, we redirect to: `http://calypso.localhost:3000/jetpack/connect/plans/complete/retail-cat.jurassic.ninja`

And I don't think there's anything on this `/jetpack/connect/plans/complete/<:site>` url that needs or makes any use of this query arg.